clustering.py

```diff
@@ -185,8 +192,43 @@ def dbscan(data):
     """
     min_max_scaler = MinMaxScaler()
     min_max_scaled = min_max_scaler.fit_transform(data)
-    dbscan_result = DBSCAN(eps=1.2, min_samples=1).fit(min_max_scaled)
-    return dbscan_result.labels_
+    dbscan_result = DBSCAN(eps=1.2, min_samples=2).fit(min_max_scaled)
+    cluster_assignment = np.copy(dbscan_result.labels_)
+    medians = cluster_medians(data, cluster_assignment)
+
+    outlier_indexes = np.where(cluster_assignment == -1)[0]
+    cluster_assignment += 1
```

Comment:
Sorry if this is my own ignorance, but what does adding 1 to an array do in Numpy?

Reply:
It adds one to each element in the array. I do this so that time series can be labeled as "-clusterLabel", which would not work with the origin labels because of -0.